src/main/java/com/mars/common/error/ErrorCode.java

```diff
@@ -33,6 +33,11 @@ public enum ErrorCode {
      * RESUME
      */
     RESUME_NOT_FOUND(404, "RS001", "존재하지 않는 이력서입니다."),
+
+    /**
+     * PORTFOLIO
+     */
+    PORTFOLIO_NOT_FOUND(404, "RS001", "존재하지 않는 포트폴리오입니다."),
```

Comment:
RS001은 이미 존재하는 코드이므로 해당 에러 코드로 특정 에러를 트래킹할 수 없습니다. (유니크한 값이 되어야함) 
해당 사항 고려해서 다른 코드로 변경해주세요!
```suggestion
    PORTFOLIO_NOT_FOUND(404, "PF001", "존재하지 않는 포트폴리오입니다."),
```